cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,96 @@
+/// <reference types='cypress' />
+
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+
+  before(() => {
+    cy.visit('/');
+    cy.task('generateUser')
+      .then(generateUser =>{
+        user = generateUser
+      });
+    });
+
+  it('check workflow on the page', () => {
```

Comment:
Inappropriate test name. Split it into different parts. You have a proper description in file readme.md, so your describe block should contain `it`s like "Add new worker", "edit worker" etc